src/ring_theory/subsemiring.lean

```diff
@@ -12,7 +12,8 @@ import data.equiv.ring
 # Bundled subsemirings
 
 We define bundled subsemirings and some standard constructions: `complete_lattice` structure,
```

Comment:
@kbuzzard You aren't interested in expanding this to a real module doc, are you? :)

Reply:
I am, but I'm about to go on holiday for four days. If you're happy to wait that long, I'll do it.

Reply:
The 80 open PRs are stressing me out so I'll just merge this now, but come back to it if you think about it. Enjoy the vacation!